package.json

```diff
@@ -15,7 +15,8 @@
     "@ministryofjustice/frontend": "1.2.0",
     "cypress": "9.4.1",
     "govuk-frontend": "^4.0.1",
-    "jquery": "^3.6.0"
+    "jquery": "^3.6.0",
+    "tom-select": "^2.0.1"
```

Comment:
Might be worth giving [accessible-autocomplete](https://alphagov.github.io/accessible-autocomplete/) a try, since it fits the gov branding better.

Edit: that said, accessible-autocomplete requires a bit more manual integration